Homework/src/io/codelex/arithmetic/practice/ExerciseNine.java

```diff
@@ -0,0 +1,40 @@
+package io.codelex.arithmetic.practice;
+
+import java.util.Scanner;
+
+//Program must accept metric units but in what way? MM, CM, DM, M? g, kg, T?
```

Comment:
Tas ir jautājums man? :) Uzdevumā bija minēts "weight is measured in pounds and height is measured in inches"